pkg/controller/core/localqueue_controller.go

```diff
@@ -204,6 +264,29 @@ func (r *LocalQueueReconciler) Update(e event.TypedUpdateEvent[*kueue.LocalQueue
 	return true
 }
 
+func (r *LocalQueueReconciler) ReconcileConsumedUsage(ctx context.Context, lq *kueue.LocalQueue, cqName kueue.ClusterQueueReference) error {
+	halfLifeTimeSeconds := r.admissionFSConfig.UsageHalfLifeTime.Seconds()
+	oldUsage := lq.Status.FairSharingStatus.AdmissionFairSharingStatus.ConsumedResources
+	// reset usage to 0 if halfLife is 0
+	if halfLifeTimeSeconds == 0 {
+		lq.Status.FairSharingStatus.AdmissionFairSharingStatus.ConsumedResources = resource.MulByFloat(oldUsage, 0)
```

Comment:
can we just set to nil (or default resource list), and get oldUsage later (right before it is used?)

Reply:
Done